dev-start-fetch.sh

```diff
@@ -1,11 +1,10 @@
 export GU_aws_profile="composer"
 export GU_s3bucket="gudocs-dev"
 export GU_s3domain="not-used"
-export GU_sns_errors="not-used"
 export GU_dbkey="gudocs-dev"
 export GU_require_domain_permissions="dev-guardian.co.uk"
 
 npm run fetch
 
 # to fetch a single file run with the id of the google doc
-# npm run fetch -i ID
\ No newline at end of file
+# npm run fetch -i ID
```

Comment:
This is the IDE adding a new line to the end of this file. This change is a no-op.